doc/husky-pre-commit.md

```diff
@@ -9,6 +9,9 @@ actions prior to every call to `git commit`:
 {! ../.husky/pre-commit extract: { start: list } !}
 ```
```

Comment:
This isn't working in the doc production:
[
![Screenshot from 2024-10-08 17-54-39](https://github.com/user-attachments/assets/5976422e-c68f-4c85-a558-51a1e11ae39a)
](url)

Reply:
Good catch. The Husky actions changed, so the inclusion directive was no longer grabbing them (its pattern wasn't matching any more). The latest commit fixes this and adds a test for it, starting a new test suite for the documentation site.